README.md

```diff
@@ -899,6 +904,87 @@ gradlew aemInstall -Paem.package.skipDownloadName=false
 ```
 Only matters when Vault properties file is customized then that property could be used to eliminate conflicts.
 
+## Contributing
+
+### Basics
+
+* Work on fork of this repository - In the top-right corner of this page, click Fork.
+* All your changes should be created from and merged into **develop** branch.
+* If you want your changes to be included in upstream release Create Pull Request from your fork **develop** branch into upstream **develop** branch
```

Comment:
To include changes into repository and make them a part of next release, it is needed to create pull request in forked repository into develop branch of base Gradle AEM Plugin repository.